examples/apps/presence-tracker/src/MouseTracker.ts

```diff
@@ -6,7 +6,8 @@
 import { Signaler } from "@fluid-experimental/data-objects";
 import { TypedEventEmitter } from "@fluid-internal/client-utils";
 import { IEvent } from "@fluidframework/core-interfaces";
-import { IMember, IServiceAudience } from "fluid-framework";
+import type { ITinyliciousAudience } from "@fluidframework/tinylicious-client/internal";
```

Comment:
I think service-audience data might be an exception to service agnosticism, since we market it as a service-specific feature to customers (https://fluidframework.com/docs/build/audience/#service-specific-audience-data) 

I'll look into ways to not use /internal APIs. However, I'm not sure these examples are to demonstrate what external customers should/could do. I remember asking Matt @ChumpChief so maybe he can help here, but I think these serve as mostly for testing(?) I could be wrong though. /internal is used a lot here so it's confusing, I think FluidExamples repo is more geared for external customer usage.

Reply:
Correct that the audience is not service-agnostic (though it would be nice if we could make it so).

I think it could be reasonable to use IAzureAudience here instead, as we typically talk about Tinylicious as being the "Azure local service" and support customers writing code to run equally against Tinylicious/Azure.  Maybe ITinyliciousAudience should go away even in favor of IAzureAudience (at least if we're not going to make it public)?

Reply:
OK thanks, I'll switch the example to use IAzureAudience. 

Is there a reason why [AzureUser](https://github.com/microsoft/FluidFramework/blob/adc603967b9ae8dfe0942049c101165f50b2aecb/packages/service-clients/azure-client/src/interfaces.ts#L147C1-L162C1) is marked as internal but [IUser](https://github.com/microsoft/FluidFramework/blob/main/common/lib/protocol-definitions/src/users.ts) is public?

Reply:
I'm not sure offhand why IUser is public, but I think in general AzureMember is what should show up on the public interface (whereas AzureUser is more internal-facing).  I'm a little fuzzy on this area though.